sdk/resourcemanager/resources/armdeployments/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 0.1.0 (2025-07-23)
+## 0.1.0 (2025-07-24)
 ### Other Changes
 
 The package of `github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/resources/armdeployments` is using our [next generation design principles](https://azure.github.io/azure-sdk/general_introduction.html).
```

Comment:
This package is split from `github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/resources/armresources`.